src/logging.rs

```diff
@@ -159,7 +176,7 @@ where
     //
     // Maximum data size is system dependent, thus this always tries the fast path and
     // falls back to the slow path if the former fails with `EMSGSIZE`.
-    let fast_res = sock.send_to(&data, SD_JOURNAL_SOCK_PATH);
+    let fast_res = sock.send(&data);
```

Comment:
I seem to remember there was a reason behind keeping this socket unconnected and using a destination address, but I can't find the details right now.
I'll get back when I managed to re-surface those, but in the meanwhile please consider sticking to the previous unconnected approach.

Reply:
Sure I can.

Reply:
Done